Sources/SpeziOnboarding/OnboardingFlow/IllegalOnboardingStepView.swift

```diff
@@ -10,7 +10,7 @@ import SwiftUI
 
 
 /// The `IllegalOnboardingStepView` is shown when the application navigates to an illegal ``OnboardingStep``.
-/// This behavior shouldn't occure at all as there are lots of checks performed within the ``OnboardingNavigationPath`` that prevent such illegal steps.
+/// This behavior shouldn't occur at all as there are lots of checks performed within the ``OnboardingNavigationPath`` that prevent such illegal steps.
```

Comment:
Thanks for fixing my spelling mistakes, can we keep that in the future? 😉

Reply:
I’m just doing what my spelling checker tells me to do. 🐑
I think Xcode has a basic one as well 🙊

Reply:
Oh wow, didn't know that, that is turned off by default, right?

Reply:
It is. Super hidden in some nested menu. It’s also not super good but still helpful.